zulipterminal/core.py

```diff
@@ -301,6 +302,9 @@ def popup_with_message(self, text: str, width: int) -> None:
         self.show_pop_up(NoticeView(self, text, width, "NOTICE"), "area:error")
 
     def show_about(self) -> None:
+        terminal_size = shutil.get_terminal_size()
```

Comment:
You changed to using shutil here - was there as a specific reason for this?

Reply:
@Swarnim114 To confirm, this looks good, but I wasn't sure why you switched to this.

Reply:
I thought it would be just simpler and give me no issues..

Reply:
`os.get_terminal_size()` works fine, no? 
I tried to replicate what @Swarnim114 did in this PR but without using `shutil` and it's working for me.

![Screenshot 2024-10-29 115454](https://github.com/user-attachments/assets/8aaebb70-2272-43fa-88f0-88d4583cd19c)

@Swarnim114 are there any shortcomings of doing it this way?